streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -102,8 +103,9 @@ public void completeRestoration() {
     }
 
     @Override
-    public void suspend() {
+    public Map<TopicPartition, OffsetAndMetadata> suspend() {
```

Comment:
On `suspend()` and `prepareCommit()` we don't commit yet, but return the offsets that need to be committed